windows_install_dependencies.bat

```diff
@@ -13,24 +13,54 @@
 :: See the License for the specific language governing permissions and
 :: limitations under the License.
 ::
+
+:: %1 First parameter is the --output_user_root value on c:\ drive - bazel uses this, we want to install dependencies per build there
+:: %2 Second parameter is the --expunge flag - when set to 1 we will force reinstall c:\opt dependencies - default 0
 @echo on
 setlocal EnableExtensions DisableDelayedExpansion
 :: Need to set shorter build paths for bazel cache for too long commands in mediapipe compilation
 :: We expect a first script argument to be "PR-1234" number passed here from jenkins so that a tmp directory will be created
-set "BAZEL_SHORT_PATH=C:\%1"
+if "%1"=="" (
+  echo No argument provided. Using default opt path
+  set "in_path=opt"
+) else (
+  echo Argument provided: Using install path %1
+  set "in_path=%1"
+)
+if "%2"=="" (
+  echo No argument provided. Using default force = 0
+  set "force=0"
+) else (
+  echo Argument provided: Using force = %2
+  set "force=1"
+)
+set "BAZEL_SHORT_PATH=C:\%in_path%"
```

Comment:
Maybe let's use names as described in parameters?
`in_path -> output_user_root`
`force -> expunge`